AirCasting/SessionViews/ChartViewModel.swift

```diff
@@ -0,0 +1,96 @@
+// Created by Lunar on 30/09/2021.
+//
+
+import Foundation
+import Charts
+
+final class ChartViewModel: ObservableObject {
+    @Published var entries: [ChartDataEntry] = []
+    
+    private let stream: MeasurementStreamEntity
+
+    private var timeUnit: Double {
+        stream.session.type == .mobile ? 60 : 60 * 60
```

Comment:
I personally prefer to have any numeric constants named. We know 60 is a minute and 60*60 is an hour but it is not visible from the first sight. Also, should we ever have to deal with other intervals, it will be harder to alter

Reply:
Since we're dealing with those across the whole codebase maybe we can make an extension on TimeInterval with those constants instead of just local ones?
```Swift
extension TimeInterval {
    static let minute: TimeInterval = 60
    static let hour: TimeInterval = minute * 60
    static let day: TimeInterval = hour * 24
    //etc, as needed
}
```

Reply:
I like this idea